common/djangoapps/student/models.py

```diff
@@ -3445,11 +3445,13 @@ class CourseEnrollmentCelebration(TimeStampedModel):
     """
     enrollment = models.OneToOneField(CourseEnrollment, models.CASCADE, related_name='celebration')
     celebrate_first_section = models.BooleanField(default=False)
+    celebrate_weekly_goal = models.BooleanField(default=False)
 
     def __str__(self):
         return (
-            '[CourseEnrollmentCelebration] course: {}; user: {}; first_section: {};'
-        ).format(self.enrollment.course.id, self.enrollment.user.username, self.celebrate_first_section)
+            '[CourseEnrollmentCelebration] course: {}; user: {}; first_section: {}; weekly_goal: {}'
+        ).format(self.enrollment.course.id, self.enrollment.user.username, self.celebrate_first_section,
+                 self.celebrate_weekly_goal)
```

Comment:
Maybe this is was a bad idea from the get go? Not super important either way. Mostly just used in the admin page.

Reply:
I dropped the values of the celebrate columns. If the only spot it really shows is in the admin page, you can already see all the info there anyway